sdk/tests/conformance/rendering/bind-framebuffer-flush-bug.html

```diff
@@ -0,0 +1,149 @@
+<!--
+Copyright (c) 2020 The Khronos Group Inc.
+Use of this source code is governed by an MIT-style license that can be
+found in the LICENSE.txt file.
+-->
+
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<link rel="stylesheet" href="../../resources/js-test-style.css"/>
+<script src="../../js/js-test-pre.js"></script>
+<script src="../../js/webgl-test-utils.js"></script>
+<script>
+"use strict";
+const wtu = WebGLTestUtils;
+
+function runTest() {
+    let canvas = document.createElement('canvas');
+    document.body.appendChild(canvas);
+    const gl = wtu.create3DContext(canvas);
+    if (!gl) {
+        testFailed('could not create context');
+        return;
+    }
+    const width = 100;
+    const height = 100;
+
+    debug('Setup');
+    canvas.width = width;
+    canvas.height = height;
+
+    const texture = gl.createTexture();
+    gl.bindTexture(gl.TEXTURE_2D, texture);
+    gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_MAG_FILTER, gl.NEAREST);
+    gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_MIN_FILTER, gl.NEAREST);
+    gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_WRAP_S, gl.CLAMP_TO_EDGE);
+    gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_WRAP_T, gl.CLAMP_TO_EDGE);
```

Comment:
nit: is it actually necessary to set the texParameters? I'd just omit them if they're not needed.

Reply:
I'm pretty sure they're needed because otherwise the texture is incomplete.